en-US/win10/ArduinoWiringPortingGuide.md

```diff
@@ -0,0 +1,356 @@
+
+{% highlight C++ %}
+
+pinMode( 29, OUTPUT );
+digitalWrite( 29, HIGH );
+
+{% endhighlight %}
+
+or
+
+{% highlight C++ %}
+
+pinMode( GPIO_5, OUTPUT );
+digitalWrite( GPIO_5, HIGH );
+
+{% endhighlight %}
+
+The pre-defined pin names can be found in `PinNumbers.h` inside any Arduino Wiring project, but since there will be different physical connector pins available depending on the hardware setup you are building for, we've also included a table here to describe which pin names are available for each device.
+
+#### Raspberry Pi 2
+
+[Pinout Diagram]({{site.baseurl}}/images/PinMappings/RP2_Pinout.png)
+
+{:.table.table-bordered}
+| Pin Define      | Corresponding Pin Number   |
+| -------------| ------------- |
+| LED_BUILTIN | *onboard LED* |
+| GPIO_* _where * refers to [0, 27]_  | *refer to pinout diagram* |
+| GPIO_GCLK | 7 |
+| GPIO_GEN* _where * refers to [0, 5]_ | *refer to pinout diagram* |
+| I2C_SCL1 | 5 |
+| I2C_SDA1 | 3 |
+| SPI_CS0 | 24 |
+| SPI_CS1 | 26 |
+| SPI_CLK | 23 |
+| SPI_MISO | 21 |
+| SPI_MOSI | 19 |
+| RXD0 | 10 |
+| TXD0 | 8 |
+
+#### Minnowboard Max
+
+[Pinout Diagram]({{site.baseurl}}/images/PinMappings/MBM_Pinout.png)
+
+{:.table.table-bordered}
+| Pin Define      | Corresponding Pin Number   |
+| -------------| ------------- |
+| GPIO_* _where * refers to [0, 9]_  | *refer to pinout diagram* |
+| I2C_SCL | 13 |
+| I2C_SDA | 15 |
+| SPI_CS0 | 5 |
+| SPI_SCLK | 11 |
+| SPI_MISO |7 |
+| SPI_MOSI | 9 |
+| UARAT1_CTS | 10 |
+| UARAT1_RTS | 12 |
+| UARAT1_RX | 8 |
+| UARAT1_TX | 6 |
+| UARAT2_RX | 19 |
+| UARAT2_TX | 17 |
+
+<a name="port_serial"></a>
+
+### Removing References to "Serial"
+
+Many Arduino sketches use `Serial` to print data to the serial console (if opened) or to write to the serial lines (USB or tx/rx). We've provided a "Log" function which will print a WCHAR* type (ascii strings or wide character strings) to the output console in Visual Studio. If you are copying a sketch built for an Arduino, you'll need to replace any of these Serial references in the Windows IoT version of the sketch.
+
+In the table below, replace the Arduino API Serial reference with the syntax in the Windows IoT column. If an API should be removed entirely, you'll see *remove* in the Windows IoT column.
+
+{:.table.table-bordered}
+| Arduino API syntax      | Windows IoT syntax   |
+| -------------| ------------- |
+| Serial.begin( int )  | *remove* |
+| Serial.write( char* str )     | *remove* *see below     |
+| Serial.print( char* str ) | Log( str )     |
+| Serial.print( int num ) | Log( num.ToString()->Begin() )      |
+| Serial.print( int num, format fmt ) | Log( num.ToString()->Begin() )      |
+
+
+####Why remove Serial.write()?
+
+Serial.write() is typically used to send raw data over the serial lines. Windows IoT Core does not currently have UART functionality (don't worry, it's coming soon!) so these types of calls should be avoided.
```

Comment:
Are we sure we want to say "it's coming soon!" here? Feel like it might be good to avoid, in case something happens and that commitment falls through.